qos-monitor/src/main/java/eu/arrowhead/core/qos/dto/event/monitoringevents/StartedMonitoringMeasurementEventDTO.java

```diff
@@ -17,4 +21,14 @@ public StartedMonitoringMeasurementEventDTO() {
 		this.eventType = QosMonitorEventType.STARTED_MONITORING_MEASUREMENT;
 	}
 
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public String toString() {
+		return new StringJoiner(", ", InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() + "[", "]")
```

Comment:
InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() is wrong here

Reply:
done